Blockzilla/SiriFavoriteViewController.swift

```diff
@@ -27,12 +27,16 @@ class SiriFavoriteViewController: UIViewController {
     }
     
     override func viewDidLoad() {
+        self.edgesForExtendedLayout = []
```

Comment:
This makes it so that when we attach a different navigation controller for the deep-linking to open this directly the view's edges start at the bottom of the navigation controller. Otherwise, the view continues to exist to the top of the super view and has parts hidden by the navigation controller. This happens because we are directly modally presenting this vc and giving it its own special, temporary nav controller.

Reply:
Happy to consider an alternative fix if you know one (or if there's one we use elsewhere in the project).